### ISSUE TITLE
App fails to build on on MacOS

### ISSUE THREAD

**Issue by reduxdj** [flutter-webrtc/flutter-webrtc#1683]
Build failure
```

ependency analysis" is unchecked. (in target 'Runner' from project 'Runner')
[   +6 ms] ** BUILD FAILED **
[        ] The following build commands failed:
[        ] 	CompileC
/Users/patricklemiuex/projects/cloudcall/jupiterui-app/build/macos/Build/Intermediates.noindex/Pods.build/Release/flutter_webrtc.build/Ob
jects-normal/x86_64/FlutterRTCMediaStream.o
/Users/patricklemiuex/.pub-cache/hosted/pub.dev/flutter_webrtc-0.12.0/macos/Classes/FlutterRTCMediaStream.m normal x86_64 objective-c
com.apple.compilers.llvm.clang.1_0.compiler (in target 'flutter_webrtc' from project 'Pods')
[        ] (1 failure)
[ +174 ms] Building macOS application... (completed in 53.0s)
[   +5 ms] "flutter macos" took 97,995ms.
[  +57 ms] Build process failed
[   +1 ms]
           #0      throwToolExit (package:flutter_tools/src/base/common.dart:10:3)
           #1      buildMacOS (package:flutter_tools/src/macos/build_macos.dart:166:5)
           <asynchronous suspension>
           #2      BuildMacosCommand.runCommand (package:flutter_tools/src/commands/build_macos.dart:62:5)
           <asynchronous suspension>
           #3      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1394:27)
           <asynchronous suspension>
           #4      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
           <asynchronous suspension>
           #5      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
           <asynchronous suspension>
           #6      FlutterCommandRunner.runCommand.<anonymous closure>
(package:flutter_tools/src/runner/flutter_command_runner.dart:372:9)
           <asynchronous suspension>
           #7      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
           <asynchronous suspension>
           #8      FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:308:5)
           <asynchronous suspension>
           #9      run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:130:9)
           <asynchronous suspension>
           #10     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
           <asynchronous suspension>
           #11     main (package:flutter_tools/executable.dart:93:3)
           <asynchronous suspension>


[ +115 ms] ensureAnalyticsSent: 114ms
[        ] Running 0 shutdown hooks
[        ] Shutdown hooks complete
[   +3 ms] exiting with code 1
```

**Platform information**
```
* **Flutter version**: Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.22.2, on macOS 14.3 23D56 darwin-arm64, locale en-US)
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 15.4)
[✓] Chrome - develop for the web
s[✓] Android Studio (version 2024.1)
[✓] VS Code (version 1.94.2)
[✓] Connected device (4 available)

[✓] Network resources
```
* **Plugin version**: 0.12
* **OS**: MacOS
* **OS version**: Sonomo 14.3

**Comment by forgandenny** [comment]
```objective-c
#if TARGET_OS_IPHONE
  if (@available(iOS 13.0, *)) {
    if(format.isMultiCamSupported != AVCaptureMultiCamSession.multiCamSupported) {
      continue;
    }
  }
#endif
 ```

**Comment by cloudwebrtc** [comment]
fixed https://github.com/flutter-webrtc/flutter-webrtc/releases/tag/0.12.0%2Bhotfix.1
